sparrowpy/classes/RadiosityFast.py

```diff
@@ -358,6 +358,80 @@ def from_polygon(
             patches_points, n_patches,
             patch_to_wall_ids)
 
+    @classmethod
+    def from_file(cls, filename: str, manual_patch_size=None,
+                       auto_walls=True):
+        """Create a Radiosity object from a blender or stl file.
+
+        Parameters
+        ----------
+        filename: string
+            file path to .blend or .stl file which should be loaded
+
+        manual_patch_size: float (default None)
+            sets the patch size for manual patch generation.
+            if None, patches are loaded automatically from the faces
+            of the CAD model.
+
+        auto_walls: bool
```

Comment:
```suggestion
        walls_from_file: bool
```
maybe more descriptive? auto could mean everything.